gtsfm/configs/onedsfm_front_end.yaml

```diff
@@ -16,20 +16,20 @@ SceneOptimizer:
       num_matched: 50
       min_score: 0.3
 
-  correspondence_generator:
-    _target_: gtsfm.frontend.correspondence_generator.det_desc_correspondence_generator.DetDescCorrespondenceGenerator
+  correspondence_generators:
+    - _target_: gtsfm.frontend.correspondence_generator.det_desc_correspondence_generator.DetDescCorrespondenceGenerator
 
-    detector_descriptor:
-      _target_: gtsfm.frontend.cacher.detector_descriptor_cacher.DetectorDescriptorCacher
-      detector_descriptor_obj:
-        _target_: gtsfm.frontend.detector_descriptor.sift.SIFTDetectorDescriptor
-        max_keypoints: 5000
+      detector_descriptor:
+        _target_: gtsfm.frontend.cacher.detector_descriptor_cacher.DetectorDescriptorCacher
+        detector_descriptor_obj:
+          _target_: gtsfm.frontend.detector_descriptor.sift.SIFTDetectorDescriptor
+          max_keypoints: 5000
 
-    matcher:
-      _target_: gtsfm.frontend.cacher.matcher_cacher.MatcherCacher
-      matcher_obj:
-        _target_: gtsfm.frontend.matcher.twoway_matcher.TwoWayMatcher
-        ratio_test_threshold: 0.8
+      matcher:
+        _target_: gtsfm.frontend.cacher.matcher_cacher.MatcherCacher
+        matcher_obj:
+          _target_: gtsfm.frontend.matcher.twoway_matcher.TwoWayMatcher
+          ratio_test_threshold: 0.8
```

Comment:
Why the extra indent?

Reply:
Because now `correspondence_generators` is a list. You'll notice an extra `-` at line 20. Each entry is an item in the list too.